bugbug/test_scheduling.py

```diff
@@ -310,7 +314,7 @@ def get_past_failures(granularity, readonly):
     elif granularity == "config_group":
         past_failures_db = os.path.join("data", PAST_FAILURES_CONFIG_GROUP_DB)
     else:
-        raise Exception(f"{granularity} granularity unsupported")
+        raise GranularityError(f"{granularity} granularity unsupported")
```

Comment:
All `GranularityError` should show the same message with an exception to the value of granularity. Thus, we could take that value as the perimeter and fill it in the error message template.

Example: 
```py
class UnexpectedGranularityError(Exception):
    def __init__(self, granularity):
        message = f"Unexpected {granularity} granularity"
        super().__init__(message)
```

Please do the same where applicable.